### ISSUE TITLE
Axios Get request stopping app without errors on specific url

### ISSUE THREAD

**Issue by dertzi** [axios/axios#2043]
**Describe the issue**
Calling axios get on this url causes app to close without errors. None of the console.logs are called nor is any error being caught.

**Example Code**
```js
async function test() {
    try {
    const data = await axios.get("https://www.bullfax.com/?q=node-quebec-politician-says-criticism-world-junior-hockey-te");
    console.log('Can we get here?');
    console.log(data);
    } catch (e) {
        console.log('Error catched');
        console.log(e);
    }
}

test();
```

**Expected behavior, if applicable**
Expected to either receive requested data or catch an error.

**Environment:**
 - Axios Version: [0.18.0]
 - Node Version: [11.9.0]
 - OS: [OSX 10.13.6]

**Comment by dylanwulf** [comment]
Looks like that URL has a circular redirect, so this is probably the same issue as #2042

**Comment by Alanscut** [comment]
hi @dertzi , have you solved this issue? If not, please try to use v0.19.0.

**Comment by chinesedfan** [comment]
Closed due to stale and lack of enough information.
